src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/bolts/RouterBolt.java

```diff
@@ -165,6 +167,9 @@ protected void handleInput(Tuple input) {
             } else if (data instanceof SubFlowsReadRequest) {
                 log.debug("Received a y-flow sub-flows request {} with key {}", data, key);
                 emitWithContext(ROUTER_TO_YFLOW_READ.name(), input, new Values(key, data));
+            } else if (data instanceof YFlowDeleteRequest) {
+                log.debug("Received a y-flow delete request {} with key {}", data, key);
+                emitWithContext(ROUTER_TO_YFLOW_DELETE_HUB.name(), input, new Values(key, data));
```

Comment:
```declarer.declareStream(ROUTER_TO_YFLOW_DELETE_HUB.name(), STREAM_FIELDS);```
`STREAM_FIELDS` defines 4 fields. 3 values are passed here. This will lead to an error.